docs/presetGuides.md

```diff
@@ -297,3 +253,129 @@ nodes:
+
+#### **`plain-custom-preset.yml`**
+```yaml
+nodes:
+    - voting: true
+      mainPrivateKey: CA82E7ADAF7AB729A5462A1BD5AA78632390634904A64EB1BB22295E2E1A1BDD
+      transportPrivateKey: 6154154096354BC3DB522174ACD8BFE553893A0991BD5D105599846F17A3383B
+      remotePrivateKey: E27AD508907524E2143EF2A3A272DDBEE7558B92550ABA5B82AD65D66B57BD00
+      vrfPrivateKey: F3C24C153783B683E40FB2671493B54480370BF4E3AB8027D4BF1293E14EB9B8
+      votingPrivateKey: EFE3F0EF0AB368B8D7AC194D52A8CCFA2D5050B80B9C76E4D2F4D4BF2CD461C1
+```
+
+Usage examples:
+
+-   Rebuild one of your Testnet/Main nodes.
+-   Create a new node from migrated opted in accounts.
+
+If your custom preset contains private keys, it's highly recommended to `encrypt` it and provide `--password` when starting or configuring the node with Bootstrap.
+
+```
+symbol-bootstrap encrypt --password 1234 --source plain-custom-preset.yml --destination encrypted-custom-preset.yml
+rm plain-custom-preset.yml
+symbol-bootstrap start --password 1234 --preset testnet --assembly dual --customPreset encrypted-custom-preset.yml
+```
+
+Remember to always keep your custom preset, the target's `addresses.yml`, and `preset.yml` privately!
+
+### Specify the Nodes' Public Keys.
+
+In addition to the above, you can provide the accounts Public Keys instead of Private keys in your custom preset.
+This adds a level of security as the private keys aren't in the custom preset. 
+Bootstrap will prompt for them when performing the different operations requiring user intervention when running the commands.
+
+Example:
+
+#### **`plain-custom-preset.yml`**
+```yaml
+nodes:
+    - voting: true
+      mainPublicKey: 93F9D99300DD04B66E727CA72667E5A2F6549A28AB8676B69ECAF30455C332E6
+      transportPublicKey: C9767496987222790518114049299DD52114BF2A8F7E5F4B70BB2B6365FAFD34
+      remotePublicKey: 2350F44F043DB9290390C59F1E1C2F06FCC1675E8B5F4CBDC00DD9DF4428E8FE
+      vrfPublicKey: 7834C8AAEFB09402DA32B22768D7CF9DCAC78A7774392E4452F07DDF776E088D
+      votingPublicKey: 4558724BE79FD0FD08A279878AF92F999B28D5622D3531CE9EFBC06466CDDF9C
+```
+
+In this case, this custom preset does not need to be encrypted as it doesn't contain any private information. Bootstrap will prompt for all private keys.
+
+```
+symbol-bootstrap start --preset testnet --assembly dual --customPreset plain-custom-preset.yml
+```
+
+**Hint:**
+
+You can provide some Private Keys, and some Public Keys in your custom preset. Remember to encrypt when Private Keys are provided!
+
+### Private Key Security Mode.
+
+By default, Bootstrap encrypts and stores Private Keys in the `./target/addresses.yml` file. This allows Bootstrap to perform the different operations by just asking for a password.
+
+It may be required that the Node cannot store the Main or any Private Key, even in encrypted form. 
+
+If this is you case, you can tell Bootstrap to not store these keys with the `privateKeySecurityMode` preset property. 
```

Comment:
```suggestion
However, it may be required that nodes do not store private keys, even in encrypted form. You can use the `privateKeySecurityMode` preset property in these cases.
```